buildSrc/src/main/kotlin/ort-application-conventions.gradle.kts

```diff
@@ -139,9 +139,10 @@ tasks.named<BuildNativeImageTask>("nativeCompile") {
     // Gradle's "Copy" task cannot handle symbolic links, see https://github.com/gradle/gradle/issues/3982. That is why
     // links contained in the GraalVM distribution archive get broken during provisioning and are replaced by empty
     // files. Address this by recreating the links in the toolchain directory.
-    val toolchainDir = options.get().javaLauncher.get().executablePath.asFile.parentFile.run {
-        if (name == "bin") parentFile else this
-    }
+    val toolchainDir = System.getenv("GRAALVM_HOME")?.let { File(it) }
+        ?: options.get().javaLauncher.get().executablePath.asFile.parentFile.run {
+            if (name == "bin") parentFile else this
```

Comment:
Is it intended to execute `if (name == "bin") parentFile else this` only if the value comes from `options.get().javaLauncher.get()` but not when it comes from `System.getenv("GRAALVM_HOME")` ?

Reply:
Yes. `GRAALVM_HOME` always points to the root of the installation by convention.